### PR TITLE
Fix content fixes

### PR DESCRIPTION
**Pull request by oxivanisher**
Fixing mostly textual things. Also removed unused about page and the online update from git.

### CODE REVIEW

File: modelreg/templates/registration/activation_complete.html [hunk 1]
```diff
@@ -4,7 +4,8 @@
 {% block content-header %}{% trans "Activation complete" %}{% endblock %}
 {% block content-main %}
     {% blocktrans %}
-    Well done, thanks!
+    Thank you! Your account is now activated and you are able to
     {% endblocktrans %}
+    <a href="{% url "auth_login" %}">{% trans "Login" %}</a>.
```
Comment (winged) [inline modelreg/templates/registration/activation_complete.html]: Please don't do this - it makes translation much, much harder, as this will generate two translation items that are not related to each other. Instead, include the <a> tag in the blocktrans.

File: modelreg/templates/registration/activation_email.txt [hunk 2]
```diff
@@ -1,18 +1,16 @@
+Hi {{user}}
```
Comment (winged) [inline modelreg/templates/registration/activation_email.txt:4]: The whole email should be wrapped in a {%blocktrans%}.

Note that to inject variables, you can do this: `{%blocktrans with domain=site.doman, url=... %} and so on.

File: modelreg/urls.py [hunk 3]
```diff
@@ -33,5 +33,4 @@
     url(r'^f/(?P<ident>[\w\d]+)\.(?P<auth>[\w\d]+)',  views.found, name='found'),
     url(r'^found/',  views.found_info, name='found_info'),
     url(r'^$',  views.index, name='index'),
-    url(r'^about/', views.about, name='about'),
```
Comment (winged) [inline modelreg/urls.py]: We decided to keep the about page, so the url should stay as well

File: modelreg/views.py [hunk 4]
```diff
@@ -112,9 +112,6 @@ def case_owner(req, pk):
 def index(req):
     return render(req, 'index.html')
 
-def about(req):
-    return render(req, 'about.html')
-
```
Comment (winged) [inline modelreg/views.py]: We need to keep this, see above :)

File: README.md [hunk 5]
```diff
@@ -83,19 +83,20 @@ using an SSL/TLS certificate. The latter is utterly important, as there will
...
-    docker build .
-    docker run
+    docker-compose up
 
-Maybe in the future, we will also provide "official" modelreg docker images,
-so you won't have to build it yourself.
+We will also provide "official" modelreg docker images which you can find [on Docker Hub](https://hub.docker.com/r/modelreg/modelreg/).
+
+To get the image up and running, you need to copy `.env.example` to `.env` and
+configure this file to your liking.
```
Comment (winged) [inline README.md:96]: We should change the env variable configuration below as well, as we're not suggesting the user to use "plain" docker

**Review by winged — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (oxivanisher) [reply README.md:96]: Yes, this is correct.

Comment (winged) [reply README.md:96]: OK as discussed offline, we'll do that in a separate PR

**Review by winged — APPROVED** [review]
